lib/web_ui/dev/test_platform.dart

```diff
@@ -195,7 +216,8 @@ To automatically create this file call matchGoldenFile('$filename', write: true)
           'y': region['y'],
           'width': region['width'],
           'height': region['height'],
-          'scale': 1, // This is NOT the DPI of the page, instead it's the "zoom level".
+          'scale':
+              1, // This is NOT the DPI of the page, instead it's the "zoom level".
```

Comment:
Undo? Looks less readable.

Reply:
I always use format document (I think others in our team also use it). In my opinion it is better to be consistent.

Reply:
Ah, I see. If we want to use dartfmt then we should probably put long comments like this on the previous line. Otherwise you may end up commenting on the wrong piece of code.